build.gradle

```diff
@@ -6,13 +6,15 @@ buildscript {
     repositories {
         google()
         gradlePluginPortal()
+        mavenCentral()
```

Comment:
`gradlePluginPortal()` pulls from both `jenter()` and `mavencentral()`.

Reply:
removed